tools/packages.config

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<packages>
```

Comment:
Ahhhaaaa. Not only removing spaces are we? :) I am not so Cakey so I don't know if it is fine. Was it even intentional?